doc/classes/CurveXYZTexture.xml

```diff
@@ -1,19 +1,26 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="CurveXYZTexture" inherits="Texture2D" version="4.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../class.xsd">
 	<brief_description>
+		A texture that shows 3 different curves (stored on the red, green and blue color channels).
 	</brief_description>
 	<description>
+		Renders 3 given [Curve]s provided to it, on the red, green and blue channels respectively. Compared to using separate [CurveTexture]s, this further simplifies the task of drawing curves and/or saving them as image files.
+		If you only need store 1 curve within a single texture, use [CurveTexture] instead. See also [GradientTexture1D] and [GradientTexture2D].
```

Comment:
```suggestion
		If you only need to store 1 curve within a single texture, use [CurveTexture] instead. See also [GradientTexture1D] and [GradientTexture2D].
```
Also I don't know if we have language guidelines on the use of 1, 2, 3 vs one, two, three. "3 curves" seems fine to me but "1 curve" looks a bit off for some pedantic reason.

Reply:
I always prefer writing numbers as digits, as it's more readable overall. That said, I agree this is debatable when it's only 1.